src/test/java/io/opentracing/contrib/jdbc/DynamicProxyTest.java

```diff
@@ -13,22 +13,41 @@
  */
 package io.opentracing.contrib.jdbc;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
 import java.util.Arrays;
+import java.util.Comparator;
 import org.junit.Test;
 
+@SuppressWarnings("all")
 public class DynamicProxyTest {
+  private static final Comparator<Class<?>> comparator = new Comparator<Class<?>>() {
+    @Override
+    public int compare(final Class<?> o1, final Class<?> o2) {
+      return o1.getName().compareTo(o2.getName());
+    }
+  };
+
+  private static void testGetAllInterfaces(final Object obj, final Class<?> ... expecteds) {
+    final Class<?>[] ifaces = DynamicProxy.getAllInterfaces(obj.getClass());
+    Arrays.sort(ifaces, comparator);
+    Arrays.sort(expecteds, comparator);
+    assertEquals(Arrays.toString(ifaces), Arrays.toString(expecteds));
+    for (final Class<?> iface : ifaces)
+      assertTrue(Arrays.stream(ifaces).anyMatch(i -> i == iface));
```

Comment:
```suggestion
      assertTrue(Arrays.asList(ifaces).contains(iface));
```

Reply:
Although this whole loop feels weird.

Reply:
Thank you for this comment! You're absolutely right! I just fixed this.